plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -471,6 +486,17 @@ public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connect
         return Optional.empty();
     }
 
+    private static LongReadFunction timestampMillisReadFunction()
```

Comment:
i understand we cannot use `mySqlTimestampReadFunction(createTimestampType(3))` because it would fail instead of round, right?

let's add "Rounding" to the function name